x-pack/plugins/security_solution/scripts/endpoint/common/stack_services.ts

```diff
@@ -188,18 +208,27 @@ export const createEsClient = ({
   password,
   apiKey,
   log,
+  noCertForSsl,
 }: {
   url: string;
   username: string;
   password: string;
   /** If defined, both `username` and `password` will be ignored */
   apiKey?: string;
   log?: ToolingLog;
+  noCertForSsl?: boolean;
```

Comment:
I like the way you base logic on an argument's default value 👍  Consider changing interface to `noCertForSsl?: true;`